student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(name, age, parentt_permission: parent_permission)
```

Comment:
- [ ] Kindly watch out for typo errors as they can throw unexpected issues. This should be `parent_permission: parent_permission` (one 't', not two)